python-checks/src/main/java/org/sonar/python/checks/VerifiedSslTlsCertificateCheck.java

```diff
@@ -165,29 +162,43 @@ private IssueReport(String message, Token token) {
     }
   }
 
+  public static final Set<String> VERIFY_ARG_NAME = Set.of("verify");
   /**
    * Set of FQNs of methods in <code>requests</code>-module that have the vulnerable <code>verify</code>-option.
    */
-  private static Set<String> requestsMethods = Stream
-    .of("request", "get", "head", "post", "put", "delete", "patch", "options")
-    .map(method -> "requests.api." + method)
-    .collect(Collectors.toSet());
+  private static final Map<String, Set<String>> FALSE_ARGUMENT_METHODS = Map.ofEntries(
+    Map.entry("requests.api.request", VERIFY_ARG_NAME),
+    Map.entry("requests.api.get", VERIFY_ARG_NAME),
+    Map.entry("requests.api.head", VERIFY_ARG_NAME),
+    Map.entry("requests.api.post", VERIFY_ARG_NAME),
+    Map.entry("requests.api.put", VERIFY_ARG_NAME),
+    Map.entry("requests.api.delete", VERIFY_ARG_NAME),
+    Map.entry("requests.api.patch", VERIFY_ARG_NAME),
+    Map.entry("requests.api.options", VERIFY_ARG_NAME),
+    Map.entry("httpx.request", VERIFY_ARG_NAME),
+    Map.entry("httpx.stream", VERIFY_ARG_NAME),
+    Map.entry("httpx.get", VERIFY_ARG_NAME),
+    Map.entry("httpx.options", VERIFY_ARG_NAME),
+    Map.entry("httpx.head", VERIFY_ARG_NAME),
+    Map.entry("httpx.post", VERIFY_ARG_NAME),
+    Map.entry("httpx.put", VERIFY_ARG_NAME),
+    Map.entry("httpx.patch", VERIFY_ARG_NAME),
+    Map.entry("httpx.delete", VERIFY_ARG_NAME),
+    Map.entry("httpx.Client", VERIFY_ARG_NAME),
+    Map.entry("httpx.AsyncClient", VERIFY_ARG_NAME)
+  );
 
   private static void requestsCheck(SubscriptionContext subscriptionContext) {
-    CallExpression callExpr = (CallExpression) subscriptionContext.syntaxNode();
-    boolean isVulnerableMethod = ofNullable(callExpr.calleeSymbol())
+    var callExpr = (CallExpression) subscriptionContext.syntaxNode();
```

Comment:
Could you maybe clarify why you choose to use `var` is this situation? I'm genuinely curious.

Reply:
I generally prefer `var` since it allows you to write shorter lines.